packages/cpp/ArmoniK.Api.Client/header/events/EventsClient.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "events_common.pb.h"
+#include "events_service.grpc.pb.h"
+
+namespace armonik {
+namespace api {
+namespace client {
+class EventsClient {
+public:
+  explicit EventsClient(std::unique_ptr<armonik::api::grpc::v1::events::Events::StubInterface> stub)
+      : stub(std::move(stub)) {}
+
+  void wait_for_result_availability(std::string session_id, std::vector<std::string> result_ids);
+
+private:
+  std::unique_ptr<armonik::api::grpc::v1::events::Events::StubInterface> stub;
```

Comment:
Same comment as before (using =...) for events namespace

Reply:
Unfortunately, we are in a header.